### ISSUE TITLE
5.1.1 upgrade wont build ios project

### ISSUE THREAD

**Issue by Johan-dutoit** [aritchie/userdialogs#166]
1) Please check all of the platforms you are having the issue on
[x] iOS
[] Android
[] UWP
[] PCL

2) Version 5.0.0-beta2 - worked perfectly
Upgrading to version 5.1.1, I get the following build error

> System.Void CoreGraphics.CGRect::.ctor(System.Single,System.Single,System.Single,System.Single)" reference from "Xamarin.iOS, Version=0.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065"

I also noticed that BTProgressHUD (1.2.0.2) was added part of the upgrade, so it could potentially be there.

Packages.config
```
 <package id="Acr.Support" version="2.0.2" targetFramework="xamarinios10" />
  <package id="Acr.UserDialogs" version="5.1.1" targetFramework="xamarinios10" />
  <package id="BTProgressHUD" version="1.2.0.2" targetFramework="xamarinios10" />
  <package id="Humanizer.Core" version="2.0.1" targetFramework="xamarinios10" />
  <package id="modernhttpclient" version="2.4.2" targetFramework="xamarinios10" />
  <package id="MvvmCross" version="4.1.4" targetFramework="xamarinios10" />
  <package id="MvvmCross.Binding" version="4.1.4" targetFramework="xamarinios10" />
  <package id="MvvmCross.Core" version="4.1.4" targetFramework="xamarinios10" />
  <package id="MvvmCross.Platform" version="4.1.4" targetFramework="xamarinios10" />
  <package id="MvvmCross.Plugin.Color" version="4.1.4" targetFramework="xamarinios10" />
  <package id="MvvmCross.Plugin.File" version="4.1.4" targetFramework="xamarinios10" />
  <package id="MvvmCross.Plugin.Messenger" version="4.1.4" targetFramework="xamarinios10" />
  <package id="MvvmCross.Plugin.Network" version="4.1.4" targetFramework="xamarinios10" />
  <package id="MvvmCross.Plugin.Visibility" version="4.1.4" targetFramework="xamarinios10" />
  <package id="MvvmCross.PortableSupport" version="3.5.1" targetFramework="xamarinios10" />
  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="xamarinios10" />
  <package id="Nito.AsyncEx" version="3.0.1" targetFramework="xamarinios10" />
  <package id="Plugin.Permissions" version="1.1.7" targetFramework="xamarinios10" />
  <package id="Splat" version="1.6.2" targetFramework="xamarinios10" />
  <package id="Xam.Plugins.Settings" version="2.1.0" targetFramework="xamarinios10" />
  <package id="Xamarin.Build.Download" version="0.2.1" targetFramework="xamarinios10" />
  <package id="Xamarin.Google.iOS.Analytics" version="3.14.0.3" targetFramework="xamarinios10" />
  <package id="Xamarin.Insights" version="1.12.3" targetFramework="xamarinios10" />
  <package id="Xamarin.TestCloud.Agent" version="0.19.0" targetFramework="xamarinios10" />
```

**Comment by stephenhauck** [comment]
I was just coming to report this as well...works on the simulator but when I target my iPad Mini running iOS 9.2.1 I get this error ...

/iOS/MTOUCH: Error MT2002: Failed to resolve "System.Void CoreGraphics.CGRect::.ctor(System.Single,System.Single,System.Single,System.Single)" reference from "Xamarin.iOS, Version=0.0.0.0, Culture=neutral, PublicKeyToken=84e04ff9cfb79065" (MT2002)

**Comment by schiari** [comment]
Same problem on my iPhone !

**Comment by aritchie** [comment]
Xamarin issue.  Using the latest RC3 from cycle 7, this issue appears to be resolved in 5.2.  If the issue persists, please do not reopen this defect.  5.2 will be pulled and the issue will be pushed to Xamarin
